pom.xml

```diff
@@ -1017,6 +1013,7 @@
             <properties>
                 <skipTests>true</skipTests>
                 <maven.javadoc.skip>true</maven.javadoc.skip>
+                <checkstyle.skip>true</checkstyle.skip>
```

Comment:
This is not really part of this change, but the semantics of `quick` really should include turning checkstyle checks off too.